### DISCUSSION TITLE
Invalid Username when getting currently playing track

### DISCUSSION THREAD

**Discussion by PxrpleDemxn** [JohnnyCrazy/SpotifyAPI-NET#819] [Q&A] [answered]
Hi, I am trying to get my currently playing track but unfortunately I am getting "Invalid Username" in console... What should I rewrite or add? Thanks
`
var config = SpotifyClientConfig.CreateDefault().WithAuthenticator(new ClientCredentialsAuthenticator("CLIENT_ID", "CLIENT_SECRET"));

        var spotify = new SpotifyClient(config);
        var current = new PlayerCurrentlyPlayingRequest(PlayerCurrentlyPlayingRequest.AdditionalTypes.Track);
        var currentlyplaying = await spotify.Player.GetCurrentlyPlaying(current);
        IPlayableItem s = currentlyplaying.Item;
        if (s is FullTrack track)
        {
            Console.WriteLine(track.Name);
        }`

**Comment by JohnnyCrazy** [comment, marked as answer]
It's not possible to use `ClientCredentialsAuthenticator` in combination with `Player.GetCurrentlyPlaying`. `ClientCredentialsAuthenticator` can not access user resources

https://johnnycrazy.github.io/SpotifyAPI-NET/docs/auth_introduction
